src/api.ts

```diff
@@ -4,12 +4,14 @@ import type {
 } from 'http';
 import { MongoDBOIDCPluginImpl } from './plugin';
 import type {
+  IdPServerInfo,
   MongoDBOIDCLogEventsMap,
   OIDCAbortSignal,
   OIDCRefreshFunction,
   OIDCRequestFunction,
   TypedEventEmitter,
 } from './types';
+import type { IntrospectionResponse, UserinfoResponse } from 'openid-client';
```

Comment:
Should these types be re-exported if they are meant to be usable from consumers of this package?